proto/regen/ecocredit/basket/v1/events.proto

```diff
@@ -4,7 +4,7 @@ package regen.ecocredit.basket.v1;
 
 import "regen/ecocredit/basket/v1/types.proto";
 
-option go_package = "github.com/regen-network/regen-ledger/x/ecocredit/basket";
+option go_package = "github.com/regen-network/regen-ledger/x/ecocredit/basket/v1";
```

Comment:
Do we want to change this now?

Reply:
if we're going to have multiple versions of proto packages, i think so. 

we kept running into an issue with overwriting due to different versioned proto packages with the same go_option, i.e. v1alpha1 and v1alpha2 both pointed to `github.com/regen-network/regen-ledger/x/ecocredit`.